.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: cpp
+dist: bionic
+jobs:
+  include:
+    - stage: Bare metal build
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-8
```

Comment:
This is needed because the Ubuntu bionic repositories don't include g++ v8. Travis CI doesn't support newer distributions at the moment.